src/angular/datepicker/datepicker/datepicker.scss

```
@@ -3,15 +3,22 @@
 .sbb-datepicker {
   display: inline-block;
   position: relative;
-  width: 100%;
   color: var(--sbb-color-weak-accent);
+
+  sbb-form-field & {
```

Nitpick (again); Use class selector unless there is a specific reason (like specificity).
```suggestion
  .sbb-form-field & {
```